CHANGELOG.md

```diff
@@ -13,11 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
-[Unreleased]: https://github.com/SumoLogic/sumologic-kubernetes-collection/compare/v2.4.0-rc.0...main
+[Unreleased]: https://github.com/SumoLogic/sumologic-kubernetes-collection/compare/v2.4.0...main
 
-## [v2.4.0_rc.0][v2_4.0_rc.0]
+## [v2.4.0][v2_4.0]
```

Comment:
The link in line 43 needs fixing :)

Reply:
Good catch! I've changed that but didn't save the file 🤦

Reply:
It's fixed now.